airtime_mvc/application/views/scripts/index/index.phtml

```diff
@@ -70,7 +70,7 @@
             <div id="jp_container_1" class="jp-audio">
                 <div class="jp-type-playlist">
 
-                    <div class="current-track track-overflow"><?php echo _("Now Playing:") ?><span class="track-overflow"></span></div>
+                    <div class="current-track track-overflow">Now Playing: <span class="track-overflow"></span></div>
```

Comment:
And the translation call here?

Reply:
Unintended.

Fixed in https://github.com/LibreTime/libretime/pull/1119/commits/b8f4a3392716b5be2ab89139cc5b65b973dec523.